src/tests/test_mem_btree.cpp

```diff
@@ -77,6 +77,8 @@ struct BtreeTest : public testing::Test {
     using K = typename TestType::KeyType;
     using V = typename TestType::ValueType;
 
+    virtual ~BtreeTest() override = default;
```

Comment:
Just `override`; we want there to be an error if the underlying removes the virtual attribute.

Reply:
I removed it. I did as you suggested, destructor for each test would be needed( which is unnecessary).